open_schools_platform/organization_management/circles/services.py

```diff
@@ -81,13 +81,14 @@ def query_wrong_recipient(query: Query, new_status: str):
 class CircleQueryHandler(BaseQueryHandler):
     allowed_statuses = [Query.Status.ACCEPTED, Query.Status.SENT, Query.Status.CANCELED, Query.Status.DECLINED]
     available_statuses: Dict[Tuple[str, frozenset], Tuple] = {
```

Comment:
I think we should make this dict:
Dict[Tuple[str, str], Tuple]
so we can write only one permission
interaction of several permission we will handle in query_handler function